src/__myproject__/_export/clickhouse_export.py

```diff
@@ -156,11 +167,11 @@ def sampled_features(df: DataFrame):
 
 # COMMAND ----------
 
-def count_percentile(col: str, percentile_percentage: float) -> Column:
-    return f.percentile_approx(f.when(f.col(col) > 0, f.col(col)), percentile_percentage)
+def count_percentile(col: str, percentile_percentage: float, accuracy: int) -> Column:
```

Comment:
In general some documentation to the whole algorithm would be nice, some comments etc..